### PR TITLE
Rework Linter

### PR DESCRIPTION
**Pull request by bh2smith**
Just an attempt to fix the issue pointed out in https://github.com/bh2smith/safe-airdrop/pull/54#discussion_r632943644

Test Plan:

```
rm -rf node_modules
rm yarn.lock
yarn
```
Restart IDE
Check that 
```
  const { addMessage, setCodeWarnings, setMessages } =
    useContext(MessageContext);
 ```
 Isn't highlighted in red or updated by `npx pretty-quick`
 
 
Note that I have also selected this in my prettier settings - which seemed to help
Screenshot 2021-05-16 at 09 03 12

I am not 100% sure what is going on here and not super satisfied that this particular line break pattern is so inconsistent with the rest. But I can live with it for now.

Note that. At some point we may have to conform to the following style guide from https://github.com/gnosis/safe-react-apps (since this project is likely headed there).

If that is the case, then everything will change a lot (cf their vastly different `.prettierrc`)

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 80,
+  "trailingComma": "es5",
+  "singleQuote": false
```
Comment (schmanu) [inline .prettierrc:4]: Should we just go with the safe-react-apps prettierrc here?
```
{
  "printWidth": 120,
  "trailingComma": "all",
  "singleQuote": true
}
```

Comment (schmanu) [reply .prettierrc:4]: Especially because the small printWidth on a large screen was always confusing to me so much wasted space.

**Comment by schmanu** [comment]
I ran the build jobs again. For some reason it failed during checkout of the repo on the last run.

File: .prettierrc (hunk 1 above)
Comment (bh2smith) [reply .prettierrc:4]: I could get behind two of the three here, I really do not like the appearance of the trailing comma on those react components. But yea, lets give it a try and see how it plays out.

Comment (bh2smith) [reply .prettierrc:4]: Its just SOOO hard to read!

```js
const web3Provider = useMemo(() => new ethers.providers.Web3Provider(new SafeAppProvider(safe, sdk)), [sdk, safe]);
```

Comment (bh2smith) [reply .prettierrc:4]: Just be aware, if we go with that prettier file, that every single project file will now be changing. I'll make a commit to show what its gonna be like.

Comment (schmanu) [reply .prettierrc]: I think its okay. But then again I never saw, that a safe apps project has to follow one specific code-style. At least its not mentioned in the contribution / grant guidelines or am I blind?

**Review by schmanu — APPROVED** [review]
I think its okay. Not a big fan of the single quotes but as long as it gets fixed automatically I'm fine with it.

**Comment by bh2smith** [comment]
I'd be happy to go back to double quotes.

**Pull request merged**